src/quo/foundations/colors.cljs

```diff
@@ -299,6 +302,30 @@
   [s]
   (and (string? s) (string/starts-with? s "#")))
 
+(defn rgb-hex?
+  [s]
+  (and (hex-string? s)
+       (= rgb-hex-length
+          (count s))))
+
+(defn hex->rgba
+  "Takes a 7 character hex color and converts it to rgb() format"
+  ([s]
+   (hex->rgba s nil))
+  ([s alpha-value]
+   (let [add-ending-parenthesis (fn [input-string]
+                                  (str input-string
+                                       (if alpha-value
+                                         (str ", " alpha-value ")")
+                                         ")")))]
+     (->> (subs s 1)
+          (partition-all 2)
+          (map #(apply (comp str (fn [number]
+                                   (js/parseInt number 16)) str) %))
```

Comment:
JS conversion from hexadecimal to decimal, Tried using `native-module.core/hex-to-number` in this ns, But component-specs didn't like it. :)